### ISSUE TITLE
DDQN isn't learning as expected

### ISSUE THREAD

**Issue by wxue24** [mendezja/Mariox#28]
<img width="622" alt="image" src="https://github.com/mendezja/Mariox/assets/64175276/007ab62f-2faa-4eda-9d0f-2da0a6dcfd5d">

The reward plot as shown above is decreasing rather than increasing over time. Could be due to hyperparameters chosen, or how the state features are preprocessed. 

Some ideas to try:
- Preprocess data (normalize values like position to between 0 and 1)
- Experiment with different neural network layers
- Experiment with different hyper parameters like "burnin", "learn_every", "sync_every", and "save_every", "batch_size", "exploration_rate"

**Comment by wxue24** [comment]
After adding preprocessing to the data and adjusting the hyperparameters I got a much better reward plot. 

<img width="569" alt="image" src="https://github.com/mendezja/Mariox/assets/64175276/35ab081d-9d8a-4b5e-9b41-4bd4469b3373">

We can try to make more adjustments to decrease the training time but it seems like the result is pretty good as the end reward is > 100.
